doc/lua_api.md

```diff
@@ -2988,27 +2947,32 @@ Elements
 
 * The label formspec element displays the text set in `label`
   at the specified position.
-* **Note**: If the new coordinate system is enabled, labels are
-  positioned from the center of the text, not the top.
+> [!NOTE]
+> If the new coordinate system is enabled, labels are positioned
+  from the center of the text, not the top.
 * The text is displayed directly without automatic line breaking,
-  so label should not be used for big text chunks.  Newlines can be
+  so label should not be used for big text chunks. Newlines can be
   used to make labels multiline.
-* **Note**: With the new coordinate system, newlines are spaced with
-  half a coordinate.  With the old system, newlines are spaced 2/5 of
-  an inventory slot.
+> [!NOTE]
+> With the new coordinate system, newlines are spaced with half a
+  coordinate. With the old system, newlines are spaced 2/5 of an inventory slot.
```

Comment:
also missing `>` here and above

Reply:
I searched for `\n> .*\n[^>*] ` and resolved all of these issues that I could find.